spec/controllers/lists_controller_spec.rb

```diff
@@ -0,0 +1,198 @@
+require 'rails_helper'
+
+begin
```

Comment:
the `begin
  ListsController
rescue
  ListsController = nil
end` portion of this code was mainly set there initially to prevent from getting strange error messages in the case where the lists controller or actions being tested in the lists controller had not been written yet. It is not really a necessary piece of code but was there just to help keep the spec test errors organized early on in the process